ios/RNCamera.xcodeproj/project.pbxproj

```diff
@@ -156,7 +156,7 @@
 		410701251ACB719800C6AA39 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastUpgradeCheck = 0620;
```

Comment:
I wonder why there is this change to `ios/.../project.pbxproj` as all the changes in this commit are related to the `/examples/basic` example ... 

Thanks a lot
I wish you a good weekend 
Fabrizio

Reply:
This happend while working with the example basic project. I saw the warning in Xcode and resolved it. I will revert the changes

Reply:
I have reverted the changes on this file.

Have a nice weekend too
René